connectivity/drivers/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-add_subdirectory(802.15.4_RF)
-add_subdirectory(ble)
-add_subdirectory(cellular)
-add_subdirectory(emac)
-add_subdirectory(lora)
-add_subdirectory(mbedtls)
+#add_subdirectory(802.15.4_RF)
```

Comment:
Why have these been commented out?

Reply:
I haven't yet got time to add all components, waiting for what we have already if it is the way we will go and I'll fix the rest

Reply:
If I do not comment out, because mbed-os target there,  it would be pulled in.